bigbluebutton-html5/private/config/settings.yml

```diff
@@ -34,6 +34,8 @@ public:
       duration: 4000
     remainingTimeThreshold: 30
     remainingTimeAlertThreshold: 1
+    showDebugWindow: false
+    showDebugWindowShortcut: true
```

Comment:
This would be better in the shortcut section:

      showDebugWindow:
        accesskey: K
        descId: showDebugWindow

Reply:
I changed the name of this parameters to make sense.
enableDebugWindowButton: (to show the button to access the debug window)
enableDebugWindowShortcut: (to enable the shortcut to show debug window)